wsgi/odyssy/basic/templates/basic/index.html

```diff
@@ -22,8 +22,18 @@
 
             <!-- Edit this section for important announcements.-->
             <div class="alert alert-warning text-center">
-                Institute invites applications for <a class="alert-link" href="http://pgadmissions.iiitvadodara.ac.in/">MTech
-                and PhD programs - Academic Year 2016-17</a>
+                {% if imp_news_announcement %}
+                    <ul id="imp_announcement">
+                        {% for announcement in imp_news_announcement %}
+                            <li>
+                                <a class="alert-link" href="#">{{ announcement.title }}:</a>
+                                <div style="margin-left: 5px">{{ announcement.description }}</div>
+                            </li>
+                        {% endfor %}
+                    </ul>
+                {% else %}
```

Comment:
So should I make the whole div of important inside `if` so that that white division block will only appear if there is anything important.